deployments/bsc.diamond.immutable.json

```diff
@@ -37,9 +37,9 @@
         "Name": "AxelarFacet",
         "Version": ""
       },
-      "0xA1Edc2eD671Dfa77eD2dCD2ee012F82e4807A75a": {
-        "Name": "CBridgeFacet",
-        "Version": ""
+      "0x3e08304D9e3feb632D2Ee027634785d51fBF9870": {
+        "Name": "CBridgeFacetPacked",
+        "Version": "1.0.1"
```

Comment:
`CBridgeFacet` and `CBridgeFacetPacked` are two different facets, both should be registered.

Reply:
Same applies to other chains.